src/common/utils/validateDuplicateFieldValue.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { validateRequired } from '@folio/stripes-acq-components';
+
+export const validateDuplicateFieldValue = async ({
+  ky,
+  api,
+  id,
+  fieldValue,
+  errorMessage,
```

Comment:
This one makes me sad - you just pass errorMessage to return it. This is not used at all in this code block. From my point of view this function could return boolean - true if there is a duplicate or false if not. Error messages could be managed at specific validation functions. This one is general and is not a form validation function per se

Reply:
However, you might leave it as is, we might be re-factor it later, when be leveraging code to the stripes-acq-components